### ISSUE TITLE
Output assets even when filtered from precache.

### ISSUE THREAD

**Issue by moimael** [goldhand/sw-precache-webpack-plugin#14]
Hi,

First thanks for the amazing plugin :)

I have an issue with it, when it comes to not using local caching of assets.

I build my static assets with webpack, and then use `staticFileGlobs: []` to prevent them to be cached locally.
I do that because once outputted, all my assets will be uploaded on amazon S3 and served via cloudfront. I then use `runtimeCaching` to cache them in the service worker.

The problem is, that  setting `staticFileGlobs: []` seems to block webpack from outputting my assets in my dist directory. Can we still output the assets while not caching them using the plugin ?

Thanks !

**Comment by goldhand** [comment]
It shouldn't be blocking webpack at all... I would like to look into this more. On another note, have you tried using `staticFileGlobsIgnorePatterns` instead of `staticFileGlobs` to prevent files from being cached?

Example: https://github.com/goldhand/sw-precache-webpack-plugin/blob/master/examples/webpack.config.js#L14

**Comment by moimael** [comment]
Hi,

After further testing it seems to happen randomly and not often, which makes me wonder if it's really an issue with this plugin. I'll monitor this in the coming weeks, see if I can find a way to reproduce.

Thanks !

**Comment by goldhand** [comment]
@moimael I'm going to close this issue unless you've continued to have trouble

**Comment by neeharv** [comment]
@goldhand I've run into the same issue consistently. Setting `staticFileGlobs` with any non empty value seems to always set `precacheConfig = []` in the generated file

**Comment by neeharv** [comment]
@goldhand in random runs, the service worker file is not output at all, and neither is there any error

eg config 

```
new SWPrecacheWebpackPlugin({
			verbose : true,
			cacheId : 'insider',
			filename : 'sw.js',
			maximumFileSizeToCacheInBytes : 4194304,
			runtimeCaching : [{
				handler : 'fastest',
				urlPattern : /^https:\/\/api\.insider\.in/,
			}],
			dontCacheBustUrlsMatching : /./,
			staticFileGlobs : [
				/\/assets\//,
				/\/built\//,
			],
		}),
```

**Comment by goldhand** [comment]
Ok I'll open this again.

**Comment by goldhand** [comment]
Should be fixed in `0.9.2`
